site/src/lib/api-client.ts

```diff
@@ -40,7 +40,7 @@ import {
 } from "../pages/api/verify-email.api";
 import { FRONTEND_URL } from "./config";
 
-const BASE_URL = `${FRONTEND_URL}/api/`;
+const BASE_URL = `${typeof window === "undefined" ? "" : FRONTEND_URL}/api/`;
```

Comment:
This small change is not related to sandbox directly, but should resolve one issue with hostnames.

When visiting a production deployment by its permalink, e.g. (https://blockprotocol-j5e8ppem1.stage.hash.ai), whoami request goes to `https://blockprotocol.org/api/me` and fails CORS:

```
Response body is not available to scripts (Reason: CORS Allow Origin Not Matching Origin)
```

Using relative path on the client should fix this problem and reduce noise on Sentry. We visit permanent deployment URLs when investigating bugs in production.